bedrock/firefox/templates/firefox/new/batm/machine-a.html

```diff
@@ -0,0 +1,73 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button with context %}
+
+{% extends "firefox/new/batm/base.html" %}
+
+{% block experiments %}
+  {% if switch('experiment-firefox-new-batm-anim', ['en-US']) %}
+    {% javascript 'experiment_firefox_new_batm_anim' %}
+  {% endif %}
+{% endblock %}
+
+{% block optimizely %}
```

Comment:
I've been assured there will be no nested tests, and the two snippets are behind separate switches so we could make sure they're never both on, but I think removing the optimizely snippet entirely will eliminate any risk. Once the traffic cop test is conclusive and we have a winning page, we can remove traffic cop and add optimizely so they can do more testing with the winner if they want to.